src/App.scss

```diff
@@ -35,3 +35,7 @@
   font-size: '14px';
   font-weight: 500;
 }
+
+.remediations-font-family {
+  font-family: 'overpass-mono', sans-serif;
+}
```

Comment:
This can be replaced by using <pre> tag instead of span with a custom class name.

Reply:
The reason I opted for the class name is there are two different layouts and only one needs the `<pre>` layout. I'm gonna keep this because I don't think it causes any issues, unless you're aware of something I'm not.